app/controllers/suppliers_controller.rb

```diff
@@ -27,6 +27,7 @@ def new
 
   def create
     @supplier = Supplier.new(supplier_params)
+    @supplier.supplier_category ||= SupplierCategory.first
```

Comment:
You don't want to leave it blank? At the beginning there is no supplier category anyway.

Reply:
i'd like to add a default category, which needs to exist, so there are no null values (i think i need to adopt the migrations to show that :roll_eyes: )

Reply:
it was in the migration already...

Reply:
oops, good job